README.md

```diff
@@ -28,10 +28,7 @@ as most LDP-based systems. It is based on decentralized OAuth2/OpenID Connect.
 
 The end result of any WebID-based authentication workflow is a verified WebID
 URI (specifically, the recipient verifies that the agent controls that URI).
-For example,
-[WebID-TLS](https://github.com/solid/solid-spec/blob/master/authn-webid-tls.md)
-derives the WebID URI from a TLS certificate, and verifies the certificate
-against the public key in an agent's WebID Profile. Similarly, the end result of
```

Comment:
This bit of info is still useful, no?
It shows that all Solid cares about is obtaining a WebID URL.

Reply:
The idea for removing this is that because TLS is no longer an aspect of the spec, this is not the right place to put this information. Perhaps a new file should be made with recommendations of various certification methods that can be implemented for your IDP. That would be the proper place to put this.

Reply:
Fair enough; probably want to create one single spec at some point (https://github.com/solid/solid-spec/issues/170)